### DISCUSSION TITLE
Ergast API

### DISCUSSION THREAD

**Discussion by sramsamy-ibm** [BrightDV/BoxBox#92] [Q&A]
Hi, 

I am trying to access ergast API but I am running into a database error. Just wondering if you have experienced this before?

Thanks

**Comment by BrightDV** [comment]
Sometimes, yes, the database is often offline after the races. However, it is a 404 error rather than a db error.
If you want to check, you can try the basic requests from the API…

I just tried to do myself a “basic request from the API” and got the same error
![image](https://github.com/BrightDV/BoxBox/assets/92821484/e6c6dd20-0c19-4c65-990b-cd6740d5d020)

I already faced the issue (often after a GP) and all you can do is wait…

**Reply by sramsamy-ibm** [reply]
Alright no worries. Thanks for the reply. I'm hoping it comes back online before tomorrow morning as thats when I get my project marked!

**Reply by BrightDV** [reply]
It seems that they updated the website. Hope it worked for you!
